supervisord/README.md

```diff
@@ -77,9 +77,9 @@ Configuration Options
 - `port` (Optional) - Defaults to 9001. The port number.
 - `user` (Optional) - Username
 - `pass` (Optional) - Password
-- `proc_names` (Optional) - Dictionary of process names to monitor
-- `server_check` (Optional) - Defaults to true. Service check for connection to the `supervisord` server.
-- `socket` (Optional) - If using `supervisorctl` to communicate with Supervisor, a socket is needed.
+- `proc_names` (Optional) - List of process names to monitor
+- `proc_regex` (Optional) - List of Regex pattern/s matching the names of processes to monitor.
+- `socket` (Optional) - Path to a UNIX domain socket on which supervisor is listening for HTTP/XML-RPC requests
```

Comment:
In most other integrations, we show a basic YAML config file, and then point users at the example config for a full listing of options. Should we move to the same style here, so as to reduce repetition of info between the README and the example config?

Reply:
Agreed

Reply:
Got it.. will make ammendments